### ISSUE TITLE
add accessibility design, need screenreader support 

### ISSUE THREAD

**Issue by kaveinthran** [GetStream/Winds#633]
I am a blind computer user, and recently tested your app. Unfortunately, your app is not accessibly design, thus it is not usable by me. Many buttons are unlabelled, there's no linear navigation and it is not keyboard navigable.Hope you will have accessibility on your highest priority.

You can test your app with this opensource screenreader https://www.nvaccess.org/download/

### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Checked that your issue isn't already filed: https://github.com/GetStream/Winds/issues

### Description

[Description of the issue]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expect to happen]

**Actual behavior:** [What actually happens]

**Reproduces how often:** [What percentage of the time does it reproduce?]

### Versions

You can get this information from the manu. Winds > About Winds.

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by httpsterio** [comment]
I can have a gander at the code base. I'm quite familiar with accessibility issues so I can try to do some PR's to see if we can get this fixed :)

**Comment by amirsol81** [comment]
Greetings. Any developments or progress regarding accessibility? I too amm a blind computer user and would very much like to be able to use Winds. However, it's not accessible with Windows screen readers -- JAWS and NVDA.
